app/src/main/java/me/saket/dank/ui/UrlRouter.java

```diff
@@ -126,10 +126,20 @@ public Intent intent(Context context) {
 
           // Smart linking.
           if (defaultWebBrowser == DefaultWebBrowser.DANK_INTERNAL_BROWSER) {
-            String packageNameForDeepLink = findAllowedPackageNameForDeepLink(url);
-            if (packageNameForDeepLink != null && isPackageNameInstalledAndEnabled(context, packageNameForDeepLink)) {
-              return Intents.createForOpeningUrl(url).setPackage(packageNameForDeepLink);
-            }
+              String packageNameForDeepLink = findAllowedPackageNameForDeepLink(url);
+              if (packageNameForDeepLink != null) {
+                  if (isPackageNameInstalledAndEnabled(context, packageNameForDeepLink))
+                      return Intents.createForOpeningUrl(url).setPackage(packageNameForDeepLink);
+                  else {
+                      /*
+                        third-party apps for only YouTube are supported at the moment.
+                        There can be more checks here if third-party apps for other services are supported too.
+                       */
```

Comment:
As per the standard Java styleguide, comments within function definitions are always single line comments:

```kotlin
// Third-party apps for only YouTube are supported at the moment.
// There can be more checks here if third-party apps for other services are supported too.
```